### ISSUE TITLE
Formatting completely broken 

### ISSUE THREAD

**Issue by Masterman467** [zixaphir/appchan-x#973]
https://i.gyazo.com/3540790099222a0a07e16cae0f20a813.png
https://i.gyazo.com/d6d3aa0313a72ddec1e7a174fbb80c1c.png

Hiroshima Nagasaki must have changed some stuff and broke appchanx. I didn't change anything sense last night when it worked fine.

**Comment by Masterman467** [comment]
It has come to my attention that today is 4/1/16 and this is probably a troll. NVM.

**Comment by zixaphir** [comment]
Yeah, I'd update, but by the time it got pushed to users, it'd already be resolved. Just disable the extension for today or live with it. Sorry.

**Comment by joelaw9** [comment]
Add this to your custom CSS. It fixes the biggest issues.
Settings > Advanced > Custom CSS

body {
  background: rgba(0,0,0,0);
}
.board {
  background: rgba(0,0,0,0);
}
div.post {
  background: rgba(0,0,0,0);
  padding:0px;
}
.board .replyContainer {
  margin:-4px;
}
div.op div.file .fileThumb img:not(.expanded-thumb), div.reply div.file .fileThumb img:not(.expanded-thumb) {
  border-radius:inherit;
  width:initial !important;
  height:initial !important;
}

**Comment by ccd0** [comment]
Have you tried just blocking
http://s.4cdn.org/css/md2016.651.css
https://s.4cdn.org/css/md2016.651.css
?

**Comment by Masterman467** [comment]
Thanks joelaw9, works well. 

Thanks for continuing to support this zixaphir. Great extension.
